### PR TITLE
#165412928 Pagination support for articles

### PR DESCRIPTION
**Pull request by djallas**
#### What does this PR do?
Pagination support for articles

#### Description of Task to be completed?
 - Using **LIMIT** and **OFFSET** to determine the article's pagination

#### How should this be manually tested?

**Route**
`/GET /api/v1/articles`

**Using query parameters**
`/api/v1/articles?limit=20&offset=0`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#165412928](https://www.pivotaltracker.com/n/projects/2326985)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: src/queries/articles/getAll.js [hunk 1]
```diff
@@ -9,13 +9,14 @@ const { Op } = Sequelize;
  * @param {object} condition condition for query
  * @returns {object} Object representing the response returned
  */
-export default async (limit = 20, offset = 0) => {
+export default async (limit, offset) => {
   const condition = { status: { [Op.ne]: 'deleted' } };
```
Comment (luc-tuyishime) [inline src/queries/articles/getAll.js]: I do not understand this part, can you help me better understand.

Comment (djallas) [reply src/queries/articles/getAll.js]: @luc-tuyishime **limit** is the number of article per page and **offset**  is used to determine the starting point to return articles rows from the database. So I removed those default values in the query to make sure that I am getting dynamic data from the controller.
Let me know things are clear or if you have a suggestion. Thanks

Comment (luc-tuyishime) [reply src/queries/articles/getAll.js]: Clear now thanks.

File: src/helpers/parameters/duplication.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @param {string} query queries to be evaluated
+ * @param {string} required
+ * @returns {array} an array of errors or an empty array if no error
+ */
+export default (query) => {
+  const { limit, offset } = query;
+  const errors = {};
+  if (Array.isArray(limit)) {
+    // errors = [...errors, 'Limit can not be declared more than once'];
```
Comment (e-liyai) [inline src/helpers/parameters/duplication.js]: @djallas delete commented out code

File: src/tests/queries/pagination/pagination.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (e-liyai) [inline src/tests/queries/pagination/pagination.js]: rename test file appropriately.

**Pull request merged**